MWFTP/manager.py

```diff
@@ -30,22 +30,24 @@ def _run_per_job(job_info: JobInfo, data_iterator_gen: Callable, xy_function: Ca
     return output_fname
 
 
-def run(job_iterator: Iterable[JobInfo], data_iterator: Callable, xy_function: Callable,
-        output_dir: str, qp_kwargs: Dict = None) -> pd.DataFrame:
+def run(job_iterator: Iterable[JobInfo], data_iterator: Callable,
+        xy_function: Callable, output_dir: str, use_fakeqp=False,
+        qp_kwargs: Dict = None) -> pd.DataFrame:
     """Creates a job for each item in job_iterator and collects the results."""
-    # os.chdir('.')
     sethandlers()
     if qp_kwargs is None:
         qp_kwargs = {}
+    qp_defaults = {'jobname': 'manager'}
 
-    with fakeqp(jobname='manager', **qp_kwargs) as q:
+    qprovider = qp if not use_fakeqp else fakeqp
+
+    with qprovider(**{**qp_defaults, **qp_kwargs}) as q:
         q.startpermanentrun()
 
         tkttores = []
         for job_info in job_iterator:
             tkttores.append(q.method(_run_per_job, (job_info, data_iterator, xy_function,
-                                                    output_dir),
-                                     _job_name=job_info.name))
+                                                    output_dir)))
```

Comment:
i think the idea here was to give every job a unique name so you can watch the progress/failures and such